extensions/map-layers-auth/src/test/ArcGisTokenGenerator.test.ts

```diff
@@ -18,7 +18,7 @@ describe("ArcGisTokenGenerator", () => {
   });
 
   it("should make proper info request and extract tokenServicesUrl from response", async () => {
-    const fetchStub = sandbox.stub(global, "fetch").callsFake(async function (_input: RequestInfo | URL, _init?: RequestInit) {
```

Comment:
Why removing the params types?

Reply:
Explicitly specifying types will cause Node types to be used for `_input`. With Node 20 types this causes build failures since Node 20 `RequestInfo` is slightly different from the web `RequestInfo`.